low-level/files/opt/seagate/sspl/bin/consuldump.py

```diff
@@ -23,7 +23,10 @@
 import argparse
 import time
 import requests
-from sspl_constants import MAX_CONSUL_RETRY, WAIT_BEFORE_RETRY, CONSUL_HOST, CONSUL_PORT, CONSUL_ERR_STRING
+import sys
+# Add the top level directories
+sys.path.insert(0, f'/opt/seagate/cortx/sspl/low-level')
```

Comment:
insert his path in python path rather than every src file

Reply:
Whenever I use sys.path.append, the new directory will be added. However, once I close python, the list will revert to the previous (default?) values.
I am working on how we can permanently add a '/opt/seagate/cortx/sspl/low-level' to PYTHONPATH.

Reply:
check
https://bic-berkeley.github.io/psych-214-fall-2016/using_pythonpath.html

Reply:
guess this comment is still open, and wherever else it applies